darts/models/forecasting/nbeats.py

```diff
@@ -166,6 +191,8 @@ def __init__(self,
             The number of fully connected layers preceding the final forking layers in each block.
         layer_width
             The number of neurons that make up each fully connected layer in each block.
+        nr_params
+            The number of parameters of the likelihood (or 1 if no likelihood is used)
         expansion_coefficient_dim
             The dimensionality of the waveform generator parameters, also known as expansion coefficients.
             Only used if `generic_architecture` is set to `True`.
```

Comment:
This also has to be updated.

Reply:
But in the `_NBEATSModule` and `NBEATSModel` classes the description does not have to be changed, because those still accept a separate parameter for the polynomial degree.